### ISSUE TITLE
Obtain pod's logs

### ISSUE THREAD

**Issue by jpkrohling** [open-telemetry/opentelemetry-demo#727]
I might be missing something, but apparently, the OTel Demo does not capture the logs produced by the services yet. I'm trying to get this working, and looks like we can use the collector helm chart to scrape logs from the node by using this:

```yaml
opentelemetry-collector:
  mode: daemonset
  presets:
    logsCollection:
      enabled: true
      includeCollectorLogs: false
```

This would break traces (see https://github.com/open-telemetry/opentelemetry-helm-charts/pull/623), but I'm sure we can come up with a workaround if the linked PR isn't the right solution.

My question: once the linked PR is merged, or when a similar solution is achieved, would this be an acceptable change to the demo? Or is there some other blocker that I'm not seeing?

**Comment by puckpuck** [comment]
The demo's logs are console output from the various components and not actual OpenTelemetry logs emitted using the Otel SDKs. The use case you present to use the collector to scrape the logs from each pod was not a use case we discussed in our SIG meetings thus far. It's certainly a valid use case, and we should investigate how we would approach and enable this in the demo.

**Comment by austinlparker** [comment]
Since we have OpenSearch available now, this is still germane. Would love help on it!

**Comment by enesonus** [comment]
I would like to work on this issue if its still available

**Comment by puckpuck** [comment]
We have converted several services to emit OTLP logs. We should create a new issue with a list of remaining services that need to be moved and track it from there.

**Comment by puckpuck** [comment]
Closed this in favor of tracking via #1472
